lib/addons/p5.dom.js

```diff
@@ -211,23 +217,21 @@ var p5DOM = (function(){
   /**
    * Creates a slider &lt;input&gt;&lt;/input&gt; element in the DOM.
    * Use .size() to set the display length of the slider.
-   * Appends to the container node if one is specified, otherwise 
+   * Appends to the container node if one is specified, otherwise
    * appends to body.
-   * 
+   *
    * @method createSlider
    * @param  {Number} min minimum value of the slider
    * @param  {Number} max maximum value of the slider
    * @param  {Number} [value] default value of the slider
-   * @param  {Number} [step] default step of the slider
    * @return {Object/p5.Element} pointer to p5.Element holding created
    *                           node
    */
-  p5.prototype.createSlider = function(min, max, value, step) {
+  p5.prototype.createSlider = function(min, max, value) {
     var elt = document.createElement('input');
     elt.type = 'range';
     elt.min = min;
     elt.max = max;
-    if (step) elt.step = step;
```

Comment:
hrm no, it looks like it reverts https://github.com/lmccart/p5.js/pull/504/files damn it. i’ll do another pr to restore.





--------------------------------------------------Chandler McWilliamshttp://brysonian.com

On Mon, Feb 2, 2015 at 7:27 AM, K.Adam White <notifications@github.com>
wrote:

>>      var elt = document.createElement('input');
>>      elt.type = 'range';
>>      elt.min = min;
>>      elt.max = max;
>> -    if (step) elt.step = step;
> This change doesn't immediately appear to be AMD-related, was this intended?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lmccart/p5.js/pull/509/files#r23931191